config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: sqlite3
```

Comment:
This adapter is ok?
```suggestion
  adapter: postgresql
```

Reply:
enconding: unicode

Reply:
Sqlite is ok for a gem test, let's avoid to add extra development dependencies if they are not needed.